File: eCopy.Model/Requests/PersonRequest.cs
using eCopy.Model.Enum;
using System;

namespace eCopy.Model
{
    public class PersonRequest
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MiddleName { get; set; }
        public Gender Gender { get; set; }
        public int? CityId { get; set; }
        public string Address { get; set; }
        public DateTime BirthDate { get; set; }
       
    }
}

File: ecopymobile/myapp/lib/model/registrationModel/personRequest.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'personRequest.g.dart';

@JsonSerializable()
class PersonRequest {
  String? firstName;
  String? lastName;
  String? middleName;
  int? gender;
  int? cityId;
  String? address;
  DateTime? birthDate;

  PersonRequest() {}

  factory PersonRequest.fromJson(Map<String, dynamic> json) =>
      _$PersonRequestFromJson(json);

  /// Connect the generated [_$PersonRequestToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$PersonRequestToJson(this);
}
